integration_test/common.js

```diff
@@ -0,0 +1,443 @@
+const ConversationPage = require('./page-objects/conversation.page');
+const { exec } = require('child_process');
+const url = require('url');
+const http = require('http');
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.Assertion.includeStack = true;
+
+const STUB_SNODE_SERVER_PORT = 3000;
+const ENABLE_LOG = false;
+
+module.exports = {
+  /* **************  USERS  ****************** */
+  TEST_MNEMONIC1:
+    'faxed mechanic mocked agony unrest loincloth pencil eccentric boyfriend oasis speedy ribbon faxed',
+  TEST_PUBKEY1:
+    '0552b85a43fb992f6bdb122a5a379505a0b99a16f0628ab8840249e2a60e12a413',
+  TEST_DISPLAY_NAME1: 'integration_tester_1',
+
+  TEST_MNEMONIC2:
+    'guide inbound jerseys bays nouns basin sulking awkward stockpile ostrich ascend pylons ascend',
+  TEST_PUBKEY2:
+    '054e1ca8681082dbd9aad1cf6fc89a32254e15cba50c75b5a73ac10a0b96bcbd2a',
+  TEST_DISPLAY_NAME2: 'integration_tester_2',
+
+  /* **************  OPEN GROUPS  ****************** */
+  VALID_GROUP_URL: 'https://chat.getsession.org',
+  VALID_GROUP_URL2: 'https://chat-dev.lokinet.org',
+  VALID_GROUP_NAME: 'Session Public Chat',
+  VALID_GROUP_NAME2: 'Loki Dev Chat',
+
+  /* **************  CLOSED GROUPS  ****************** */
+  VALID_CLOSED_GROUP_NAME1: 'Closed Group 1',
+
+  async timeout(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  },
+
+  async startApp(env = 'test-integration-session') {
+    const app = new Application({
+      path: path.join(__dirname, '..', 'node_modules', '.bin', 'electron'),
+      args: ['.'],
+      env: {
+        NODE_APP_INSTANCE: env,
+        NODE_ENV: 'production',
+        LOKI_DEV: 1,
+        USE_STUBBED_NETWORK: true,
+        ELECTRON_ENABLE_LOGGING: true,
+        ELECTRON_ENABLE_STACK_DUMPING: true,
+        ELECTRON_DISABLE_SANDBOX: 1,
+      },
+      startTimeout: 10000,
+      requireName: 'electronRequire',
+      // chromeDriverLogPath: '../chromedriverlog.txt',
+      chromeDriverArgs: [
+        `remote-debugging-port=${Math.floor(
+          Math.random() * (9999 - 9000) + 9000
+        )}`,
+      ],
+    });
+
+    chaiAsPromised.transferPromiseness = app.transferPromiseness;
+
+    await app.start();
+    await app.client.waitUntilWindowLoaded();
+
+    return app;
+  },
+
+  async startApp2() {
+    const app2 = await this.startApp('test-integration-session-2');
+    return app2;
+  },
+
+  async stopApp(app) {
+    if (app && app.isRunning()) {
+      await app.stop();
+      return Promise.resolve();
+    }
+    return Promise.resolve();
+  },
+
+  async killallElectron() {
+    return new Promise(resolve => {
+      exec('killall -9 electron', (err, stdout, stderr) => {
+        if (err) {
+          resolve({ stdout, stderr });
+        } else {
+          resolve({ stdout, stderr });
+        }
+      });
+    });
+  },
+
+  async rmFolder(folder) {
+    return new Promise(resolve => {
+      exec(`rm -r ${folder}`, (err, stdout, stderr) => {
```

Comment:
You can use `fs-extra` here instead of relying on system commands. The system commands won't work on windows.
Ref: https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/remove.md

It might be good to also update `fs-extra` version since they have added many improvements.